src/DashboardService.Models.Dto/Requests/Group/CreateGroupRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Group;
+
+public record CreateGroupRequest
+{
+  public Guid BoardId { get; set; }
+  public string GroupName { get; set; }
+  public bool GroupIsActive { get; set; }
+  public Guid GroupCreatedBy { get; set; }
+  public DateTime GroupCreatedAtUtc { get; set; }
+  public Guid? GroupModifiedBy { get; set; }
```

Comment:
this is not sent in the request, this is set already when handling the request
in PatchGroupRequest it’s similar